io.github.maddecoder.Classic-RBDOOM-3-BFG.yml

```diff
@@ -0,0 +1,136 @@
+- libgdi.yml
+- name: DoomBFA
+  buildsystem: cmake
+  subdir: neo
+  builddir: true
+  no-make-install: true
+  config-opts:
+   - --preset=linux-retail
+   - -B .
+   - -DMAKE_DLL=OFF
+   - -DPACKAGED=ON
+   - -DUSE_VCPKG=OFF
+   - -DALL_STATIC=OFF
+   - -DFFMPEG_FOUND=true
+   - -DFFMPEG_AVCODEC_INCLUDE_DIR=/app/include
+   - -DFFMPEG_LIBAVCODEC=/app/lib/libavcodec.so
+   - -DFFMPEG_LIBAVFORMAT=/app/lib/libavformat.so
+   - -DFFMPEG_LIBAVUTIL=/app/lib/libavutil.so
+   - -DFFMPEG_LIBSWRESAMPLE=/app/lib/libswresample.so
+   - -DFFMPEG_LIBSWSCALE=/app/lib/libswscale.so
+   - -DOPENAL_FOUND=true
+   - -DOPENAL_INCLUDE_DIR=/app/include
+   - -DOPENAL_LIBRARY=/app/lib/libopenal.so
+   - -DOPENAL_VERSION_STRING=1.23.1
+   - -DUSE_SYSTEM_ZLIB=false
+   - -DZLIB_FOUND=true
+   - -DZLIB_INCLUDE_DIRS=/app/include
+   - -DZLIB_LIBRARIES=/app/lib/libz.so
+   - -DUSE_SYSTEM_LIBPNG=false
+   - -DPNG_FOUND=true
+   - -DPNG_INCLUDE_DIRS=/app/include
+   - -DPNG_LIBRARIES=/app/lib/libpng.so
+   - -DUSE_SYSTEM_LIBJPEG=false
+   - -DJPEG_FOUND=true
+   - -DJPEG_INCLUDE_DIRS=/app/include
+   - -DJPEG_LIBRARIES=/app/lib64/libjpeg.so
+   - -DSDL2_INCLUDE_DIR=/app/include/SDL2
+   - -DSDL2_LIBRARY=/app/lib/libSDL2.so;/app/lib/libSDL2main.a
+  sources:
+  - type: git
+    url: https://github.com/MadDeCoDeR/Classic-RBDOOM-3-BFG.git
+    tag: 1.3.0-BFA-PATCHED
+    commit: 9044c7b78a015325b7ead958b026fbad66643458
+    disable-submodules: true
+  modules:
+    - shared-modules/glew/glew.json
+    - shared-modules/glu/glu-9.json
+    - shared-modules/SDL2/SDL2-with-libdecor.json
+    - openal.json
+    - ffmpeg.json
+    - rapidjson.json
+  build-commands:
+    - install -Dm 755 DoomBFA /app/bin/DoomBFA
+    - mkdir -p /app/share/games/doombfa
+    - cp -R ../../base /app/share/games/doombfa/base
+    - cp -R ../../base_new /app/share/games/doombfa/base_NEW
+
+- name: CDL
+  buildsystem: simple
+  sources:
+  - type: archive
+    url: https://github.com/MadDeCoDeR/CRBDL/releases/download/1.0.1.6-21/CDL.zip
+    sha512: d4866b7fc1b244a8bfd30c61ff6cbccf1db0f5b05e7506741c70c02cabddc52b67fab5d872ba1dd8992135d1930e4b96e89e073e4a381fbbd1202bbc073a7e1e
+  - type: file
+    path: install-mono.sh
+  build-commands:
+  - install -Dm 755 CDL.exe /app/bin/CDL.exe
+  - ./install-mono.sh
+
+- name: DoomBFA-assets
+  buildsystem: simple
+  sources:
+  - type: extra-data
+    filename: BFA-Assets.zip
+    url: https://github.com/MadDeCoDeR/BFA-Assets/releases/download/1.3.0/BFA-Assets.zip
+    sha256: 42901b1024dd236a5513c77920752337f63f15fd8f4810d344a98d0133a775c3
+    size: 172365
+    x-checker-data:
+      type: json
+      url:  https://api.github.com/repos/MadDeCoDeR/BFA-Assets/releases/latest
+      tag-query: .tag_name
+      version-query: $tag
+      timestamp-query: .published_at
+      url-query: .assets[] | .browser_download_url
+  - type: file
+    path: Classic-RBDOOM-3-BFG.appdata.xml
+  - type: file
+    path: DOOMBFA.desktop
+  - type: file
+    path: doom.png
```

Comment:
there is no icon upstrean?

Reply:
There is but is the basic DOOM 3 BFG Edition icon. It might be confusing to use it

Reply:
then why not put the right one there. it's confusing indeed.

Reply:
Because is the stock DOOM 3 BFG icon. Also in the upstream is available ony as a Windows .ico file or a C file

Reply:
so the purpose of the fork is not to make it better?

Reply:
Make what better?

Reply:
The icon used here is unique and is NOT on the upstream since it doesn't use a compatible format for the engine and is vice versa for the icon on upstream. So the icon file will stay